src/js/actions/ActionsCreators.js

```diff
@@ -253,8 +253,7 @@ export function setPeriod(payload) {
 
           if (timeLeft < 120 && timeLeft > 0) {
             dispatch(Actions.showNotification({
-              message: text(`This contract can not be traded 
-          in the final 2 minutes before settlement`),
+              message: text('This contract can not be traded in the final 2 minutes before settlement'),
```

Comment:
what exactly is this change, it looks the same, is it simply removal of newline?